Tasks/c0_fib.py

```diff
@@ -15,4 +23,17 @@ def fib_iterative(n: int) -> int:
 	:param n: number of item
 	:return: Fibonacci number
 	"""
-	return None
\ No newline at end of file
+	if n < 0 or type(n) == float:
+		raise ValueError
+	if n == 1:
+		return 0
+	if n == 2:
+		return 1
+	i = 1
+	first_n, second_n = 0, 1
+	number = 0
+	while i < n:
```

Comment:
```suggestion
	for _ in range(n):
```
Все таки вы знаете количество шагов, более того, сама переменная вам не нужна.
Неблокирующий